proto/minder/v1/minder.proto

```diff
@@ -3271,6 +3271,23 @@ message ListEvaluationHistoryRequest {
     // Timestamp representing the end time of the selection window.
     google.protobuf.Timestamp to = 9;
 
+    // Filter evaluation history to only those matching the specified labels.
+    //
+    // The default is to return all user-created profiles; the string "*" can
+    // be used to select all profiles, including system profiles.  This syntax
+    // may be expanded in the future.
+    repeated string label_filter = 11 [
```

Comment:
Should this be a single string, similar to what the evaluation history and list profiles requests have?

Reply:
Filters usable in the history-list endpoint are all repeated string so that they can be sent (e.g. via a REST GET) as separate entries (e.g. `?label_filter=foo&label_filter=bar`), as comma separated strings (e.g. `?label_filter=foo,bar`), or as a combination of the two.
The handler is responsible for splitting the string and populating the struct input to the service/db layer as an array of strings. Layers outside of the control plane only reason in terms of array of strings.

My understanding of `label_filter` in Profiles is that it can be only passed as [comma separated string](https://github.com/mindersec/minder/blob/main/internal/db/domain.go#L58), which is supported in this case as well. Additionally, the database layer is responsible for splitting the string, while here the responsibility is separate, which hopefully makes lower interfaces clearer.

TL;DR: it already support comma separated strings of labels, I would keep it this way.